api/src/main/resources/jakarta/servlet/resources/web-app_5_1.xsd

```diff
@@ -0,0 +1,342 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsd:schema xmlns="http://www.w3.org/2001/XMLSchema"
+            targetNamespace="https://jakarta.ee/xml/ns/jakartaee"
+            xmlns:jakartaee="https://jakarta.ee/xml/ns/jakartaee"
+            xmlns:xsd="http://www.w3.org/2001/XMLSchema"
+            elementFormDefault="qualified"
+            attributeFormDefault="unqualified"
+            version="5.1">
+  <xsd:annotation>
+    <xsd:documentation>
+
+      Copyright (c) 2009, 2020 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Do we still need this line in the new versions? It is based on the existing work so I am guessing yes.

Reply:
I'm fairly sure we need to keep that.

Btw, when changing over to 5.1, let's be extra careful TCK jobs etc run against the 5.0 branch, which could still be updated (the release ballot hasn't started yet)